test-userdata.yaml

```diff
@@ -4,7 +4,7 @@ application_id: my-http-test-app-ami
 application_version: "1.1"
```

Comment:
version should be dynamic, something like that
```application_version: "{{Arguments.ImageVersion}}"```
(and be registered in yourturn)